documentation/grammars/en_toy/verbalize/decimal.grm

```diff
@@ -0,0 +1,28 @@
+import '../byte.grm' as b;
+import '../util.grm' as u;
+import 'numbers.grm' as n;
+
+q = u.q;
+
+# Used to allow for different numbers of spaces coming out of the serializer.
+s = u.s;
+
+d = b.kDigit;
+
+
+# Protobuf: decimal { integer_part: "30" fractional_part: "056" }
+
+
+decimal_all =
```

Comment:
I think we want to do multiple verbalizations for decimals, e.g.

2.50 could be:
two point five zero
two fifty
two point five
So we can just add all of those to this file, and take union (if we were using in sparrowhawk we would add weights to ones we prefer less)